packages/std/schema/init_result.json

```diff
@@ -1,25 +1,25 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "InitResult",
-  "anyOf": [
+  "oneOf": [
     {
       "type": "object",
       "required": [
-        "ok"
+        "Ok"
       ],
       "properties": {
-        "ok": {
+        "Ok": {
           "$ref": "#/definitions/InitResponse_for_NoMsg"
         }
       }
     },
     {
       "type": "object",
       "required": [
-        "err"
+        "Err"
```

Comment:
So the only real change to serialization is that `ok` and `err` are capitalized now?

Reply:
Right